### ISSUE TITLE
Embedded Application Parameter Not Working

### ISSUE THREAD

**Issue by mjones12209** [awslabs/amazon-quicksight-embedding-sdk#137]
I am having issues with the embed SDK in a react application where we are embedding a dashboard and passing parameters like so:

![image](https://user-images.githubusercontent.com/61482297/205745705-948bcdfa-2235-4490-96a5-aa8ce76c4048.png)

The dashboard I am having issues with consists of an anomaly analysis and two line charts. The anomaly analysis does not work correctly when embedded in the react application. The expected behavior is that it applies the organization parameter to filter by organization and the current behavior is that the parameter is not applied to the filter at all. However, when we create two separate dashboards with static default parameters in the AWS Quicksight console, the anomaly analysis works fine.

**Comment by jiajing28** [comment]
When testing through the AWS QuickSight console, did the anomaly analysis respect parameter value changes through control changes or url parameter changes? If possible, can you test these setups and let us know your results.

**Comment by outspokeio** [comment]
I'll tap in here (part of the same team as @mjones12209).

When trying to change parameters dynamically in the QuickSight console it does not seem to work. However, it will work with the default parameter. Currently the only viable solution we can find is to create a separate Dashboard for each user, setting the default parameter value to match the user's "Company ID" and then creating a new dashboard from the same analysis.

Once we've done this, we have two different Dashboards showing different results that appear to be tied to the unique Company IDs.'

To be clear, every other visual works fine w/ dynamic parameters and those passed from this package. It's only the anomaly detection visual that seems to have a problem with it.

**Comment by sonals-amazon** [comment]
@outspokeio We created a dummy dashboard on our side with some anomaly detection and we did not see this issue either on the QuickSight console or via Embedding.

Since a default value is working I suspect the issue is with how the parameter value is being passed. On the website did you try adding a control and then passing value? Does that work? Or are you trying to pass parameter value using url?
